src/utils/arrayUtils.js

```diff
@@ -0,0 +1,15 @@
+/*
+Function that receive a list and a index to remove a element list.
+Function find the element to remove by id, and then create a new list without the element.
+if cant find the element, function return original list
+Params:
+  list: array of objects
+  id: number
+*/
+
+export const removeArrayElementById = (list, id) => {
+  const elementIndex = list.findIndex(element => element.id === id);
```

Comment:
I'd receive this arrow function via parameter, and define this comparator as a default.

Reply:
If the method is called removeArrayElementById I think this is ok.